imageroot/pypkg/get_route.py

```diff
@@ -35,6 +35,10 @@ def get_route(data):
     route = {}
     api_path = os.environ["API_PATH"]
 
+    agent_id = os.getenv("AGENT_ID", "")
+    if not agent_id:
+        raise Exception("AGENT_ID not found inside the environemnt")
+
```

Comment:
IIUC the "Pythonic way" it's easier, more readable and "debuggable" to (1) not provide a default and check the return value, (2) let the exception raise (3) handle it somewhere, eventually. 

In this case it's unlikely that AGENT_ID is not set in the environment: that would be a major issue. If that happens anyway the default handler is enough for us. 

The error message and the stack trace would help to find the problem.

```suggestion
    agent_id = os.environ["AGENT_ID"]
```